mono/metadata/boehm-gc.c

```diff
@@ -105,7 +105,9 @@ mono_gc_warning (char *msg, GC_word arg)
 }
 
 static void on_gc_notification (GC_EventType event);
-static void on_gc_heap_resize (size_t new_size);
+
+// GC_word here to precisely match Boehm. Not size_t, not gsize.
+static void on_gc_heap_resize (GC_word new_size);
```

Comment:
why not GC_on_heap_resize_proc?

Reply:
That is a less common style but yes, if it works, good.

Reply:
Yes should work I'll push that.

Reply:
Actually no it won't work, because `GC_on_heap_resize_proc` includes pointer.
It'd work if there was another typedef w/o the pointer.